Makefile

```diff
@@ -8,5 +10,37 @@ fmt:
 vet:
 	go vet ./...
 
+build-image:
+	${CONTAINER_RUNTIME} build -t ${IMG} .
+
 cmd: fmt vet
 	go build -ldflags="-w -s" -o bin/addon github.com/konveyor/tackle2-addon-windup/cmd
+
+.PHONY: test-e2e
```

Comment:
I think you are going to want each of start-minikube, install-tackle, and test-e2e to be `.PHONY`, put the line just above the target.